constraints/command.js

```diff
@@ -0,0 +1,486 @@
+//Modifications:
+//8-28-2020 Edward Shingler created functions ["notMutualCommand","dominates","pairExists","areAdjacent"] and constraints ["ccPhi","antiCCPhi","mutualSplit"]
+//These constraints take an boolean argument called "adjacent" defaulted to false. If true, then each function only looks at adjacent words that would cause violations.
+
+//DON'T FORGET TO INCLUDE ADJACENCY FOR ccPhi, antiCCPhi, mutualSplit
+function isInArray(myArray, x)
+{
+	var answer = false;
+	for(var i = 0; i < myArray.length; i++)
+	{
+		if(x == myArray[i])
+		{
+			answer = true;
+		}
+	}
+	return answer;
+};
+
+function getNodes(myTree)
```

Comment:
Is this used anywhere other than in getPhis? If not, I'm not sure it's useful to have around.

Reply:
It is used in the "sharePhi" function as well